client/client-multi/client-cpp/CMakeLists.txt

```diff
@@ -245,6 +241,13 @@ if (SQLITE_LOG_STORAGE_ENABLED)
 endif()
 
 if (WIN32 AND NOT CYGWIN AND NOT MSYS)
+    if (CMAKE_SYSTEM_VERSION)
+        set(version ${CMAKE_SYSTEM_VERSION})
+        string(REGEX REPLACE "^([0-9])[.]([0-9]).*" "0\\10\\2" version ${version})
```

Comment:
`\\.` instead of `[.]`?
Use `${CMAKE_SYSTEM_VERSION}` directly (don't reassign `version`).